### PR TITLE
Update node-template  `construct_runtime!` syntax

### PR DESCRIPTION
**Pull request by NukeManDan**
Per #9681, the `construct_runtime!` section can now just specify the pallets, and no need to concretely set the generics they need for the template. 

~~I will create a new tag for this for the downstream template (`monthly-2021-11+1`)~~ 
~~I would like to **move the pre-release tag** (as the build was broken for the tag as it is now) to include it in updating it for the monthly release https://github.com/substrate-developer-hub/substrate-node-template/issues/258~~
No need to change tags, it will just be included in the template downstream :heavy_check_mark:

### CODE REVIEW

File: bin/node-template/node/src/chain_spec.rs [hunk 1]
```diff
@@ -150,5 +150,6 @@ fn testnet_genesis(
 			// Assign network admin rights.
 			key: root_key,
 		},
+		transaction_payment: Default::default(),
```
Comment (NukeManDan) [inline bin/node-template/node/src/chain_spec.rs:153]: Was this somehow missed? I could not build https://github.com/paritytech/substrate/tree/monthly-2021-11 out of the box :grimacing:

Comment (thiolliere) [reply bin/node-template/node/src/chain_spec.rs:153]: yes below in transaction payment we don't write the Config part, thus the genesis config of transaction payment is not included.
You can either do `exclude_part { Config }` or do as you did (which I think is better)

Comment (thiolliere) [reply bin/node-template/node/src/chain_spec.rs:153]: yes default is good

**Review by bkchr — APPROVED** [review]

**Review by thiolliere — APPROVED** [review]

**Comment by kianenigma** [comment]
bot merge

**Pull request merged**